File: JAVA/Clothing.java
/* Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 2 dalam mata kuliah DPBO
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... */

class Clothing extends Product{
    private String size;
    private String material;
    private char gender;
    
    Clothing(){        
    }

    public String getSize(){
        return this.size;
    }

    public void setSize(String size){
        this.size = size;
    }

    public String getMaterial(){
        return this.material;
    }

    public void setMaterial(String material){
        this.material = material;
    }

    public char getGender(){
        return this.gender;
    }

    public void setGender(char gender){
        this.gender = gender;
    }
}

File: PHP/Program/Clothing.php
<?php

class Clothing extends Product{
    private $size;
    private $material;
    private $gender;

    function __construct(){
    }

    function setSize($size){
        $this->size = $size;
    }

    function getSize(){
        return $this->size;
    }

    function setMaterial($material){
        $this->material = $material;
    }

    function getMaterial(){
        return $this->material;
    }

    function setGender($gender){
        $this->gender = $gender;
    }

    function getGender(){
        return $this->gender;
    }

    function __destruct(){
    }
}

?>